tests/test_codeblock.py

```diff
@@ -1,14 +1,35 @@
 import pytest
 from sybil.compat import StringIO
 from sybil.document import Document
+from sybil.example import Example
 from sybil.parsers.codeblock import CodeBlockParser, compile_codeblock
 from tests.helpers import document_from_sample, evaluate_region
 
 
-def test_basic():
+@pytest.fixture
+def loltest_cleanup():
+    yield
+    if 'loltest' in CodeBlockParser._LANGUAGES:
+        del CodeBlockParser._LANGUAGES['loltest']
```

Comment:
Unclear why this is here? code coverage appears to suggest the cleanup isn't needed. But, it also all goes away if we lose the class-level state and just have the language evaluator as an instance variable :-)

Reply:
It was needed, just had a typo in the string.